ui/apps/platform/src/Containers/AccessControl/AccessControl.tsx

```diff
@@ -72,34 +64,30 @@ function AccessControl(): ReactElement {
                     </>
                 }
             />
-            {hasReadAccessForAccessControlPages ? (
```

Comment:
I would have probably kept the check extending it to `hasReadAccess('Access') || hasReadAccess('Role')`

Reply:
In theory
* We could have kept this logic by replacing Access with Role (if it is true that all 4 entities need Role)
* We could have added only extra Access dependency for Auth providers page.

But if future improvement in resources means that Access is the only dependency for all 4 entities, then we will move resource dependency from container code to declarative description of routes. Therefore,
* Deletions from this page now mean no more changes required here in the future.
* Additions to other pages will need to be deleted in the future.

Reply:
I think currently(meaning until after `Access` is fused with `Role`+with changes in this PR) we get reasonable behaviour when giving user `Access` but no `Role` permission: 
1. Auth providers are visible
2. Minimum access role name is visible but impossible to access
3. It's impossible for user to create either groups or change minimum access role.